lib/creatubbles/base_collection.rb

```diff
@@ -13,9 +13,15 @@ def initialize(connection)
     @connection = connection
   end
 
-  def find(id)
+  def find(id,opts={nil_404:false})
```

Comment:
I'd prefer if we could either use a separate method or (preferably) throw a custom exception (RecordNotFound). In my mind, `find` is expected to always return an object or throw an exception otherwise.